### DISCUSSION TITLE
mpv 播放出来的声音和原 Youtube 视频不一样

### DISCUSSION THREAD

**Discussion by otakutyrant** [hooke007/mpv_PlayKit#192] [mpv 通用]
原 Youtube 视频下载到本地并用 `mpv no-config` 播放，声音听起来很怪，失真了。用 VLC 播放则就和在 Youtube 上看原视频一样正常。请问有什么好办法吗？

此外 mpv 的特点对音箱按钮很敏感，稍微扭一点点就音量变化很大。不知对问题的解决有没有帮助。

**Comment by zhongfly** [comment]
建议提供一个样本

**Comment by otakutyrant** [comment]
后来把 PulseAudio 和相关包统统卸载了，安装了 Pipewire 和 EasyEffects，这问题解决了。
